  try {
    throw new RuntimeException();
  } catch (Exception e) {
  }
